Installation_ubuntu16.md

```diff
@@ -0,0 +1,64 @@
+# 1. install nvm/node
+```
+$ cd ~/
+$ git clone https://github.com/creationix/nvm.git .nvm
+$ cd ~/.nvm 
+$ git checkout v0.33.8
+$ . nvm.sh
+$ cat >>~/.bashrc<<EOF
+export NVM_DIR="$HOME/.nvm"
+[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh" 
+[ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion"
+EOF
+$ cd ~/
+$ nvm ls-remote
+$ nvm install --lts=carbon 
+$ node -v 
+```
+
+# 2. install mongodb/rabbitmq
```

Comment:
"/" means *and*?

Reply:
yes.

Reply:
Normally `/` means "or". You should use the word "and" here.